src/Directory.Packages.props

```diff
@@ -4,20 +4,20 @@
   </PropertyGroup>
   <ItemGroup>
     <!--    add your PackageVersion nodes here -->
-    <PackageVersion Include="Microsoft.Azure.Kusto.Ingest" Version="12.1.0" />
-    <PackageVersion Include="serilog" Version="2.12.0" />
-    <PackageVersion Include="Serilog.Formatting.Compact" Version="1.1.0" />
-    <PackageVersion Include="Serilog.Formatting.Compact.Reader" Version="2.0.0" />
-    <PackageVersion Include="Serilog.Sinks.PeriodicBatching" Version="3.1.0" />
-    <PackageVersion Include="Serilog.Sinks.File" Version="5.0.0" />
-    <PackageVersion Include="Serilog.Sinks.Console" Version="4.1.0" />
+    <PackageVersion Include="Microsoft.Azure.Kusto.Ingest" Version="12.2.3" />
+    <PackageVersion Include="serilog" Version="4.0.0" />
+    <PackageVersion Include="Serilog.Formatting.Compact" Version="3.0.0" />
+    <PackageVersion Include="Serilog.Formatting.Compact.Reader" Version="4.0.0" />
+    <PackageVersion Include="Serilog.Sinks.PeriodicBatching" Version="5.0.0" />
+    <PackageVersion Include="Serilog.Sinks.File" Version="6.0.0" />
+    <PackageVersion Include="Serilog.Sinks.Console" Version="6.0.0" />
     <PackageVersion Include="System.IO" Version="4.3.0" />
     <PackageVersion Include="System.IO.FileSystem" Version="4.3.0" />
-    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
-    <PackageVersion Include="xunit" Version="2.4.2" />
-    <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.5" />
-    <PackageVersion Include="coverlet.collector" Version="3.2.0" />
-    <PackageVersion Include="FluentAssertions" Version="6.9.0" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageVersion Include="xunit" Version="2.9.0" />
+    <PackageVersion Include="xunit.runner.visualstudio" Version="2.8.2" />
+    <PackageVersion Include="coverlet.collector" Version="6.0.2" />
+    <PackageVersion Include="FluentAssertions" Version="6.12.0" />
     <PackageVersion Include="Xunit.SkippableFact" Version="1.4.13"/>
```

Comment:
branch names can be chore/Upgrade dependencies.

feature/MyNewFeature
bugFix/SomeGithubissue

etc

Reply:
Makes sense. Will keep in mind.